deployer/deployer.go

```diff
@@ -24,6 +25,16 @@ type Environment struct {
 	AWS         *environment_pb.AWS
 }
 
+type DeploymentQuerier struct {
+	storage DeployerStorage
+
+	*deployer_spb.UnimplementedDeploymentQueryServiceServer
+}
+
+func NewDeploymentQuerier(storage DeployerStorage) *DeploymentQuerier {
+	return &DeploymentQuerier{storage: storage}
+}
+
 type DeploymentManager struct {
```

Comment:
I was following a bit more of only what was present here when coming up with `manager` and `querier`, but it does beg the question whether we're really just heading towards a `service` and a `worker` and calling them such?